torch/_inductor/codegen/debug_utils.py

```diff
@@ -10,20 +11,24 @@
 
 
 class DebugPrinterManager:
+    DEBUG_FILTER_DEFAULT_PRINT_ALL = "default"
+
     def __init__(
         self,
         enable_debug_printer: bool,
-        args_to_print: List[str],
-        kernel_name: str,
+        args_to_print: List[str] | None = None,
```

Comment:
better to do `Optional[List[str]]` to be consistent with type hints

Reply:
will do this minor update along in the follow-up PR.